components/View/RawTable/Headers/Header/index.js

```diff
@@ -74,9 +74,9 @@ export default class Header extends React.PureComponent {
         );
 
         const props = {
-            hoverable,
-            highlighted,
-            uniqueKey,
+            hoverable: hoverable.toString(),
+            highlighted: highlighted.toString(),
+            key: uniqueKey,
```

Comment:
Why pass key to a single child. Also, we shouldn't remove "uniqueKey" prop that the children may be using.
Also, converting the boolean to string doesn't do much than change the api.

Looks like there is problem with the children instead of this component.